nlp/src/main/java/hivemall/nlp/tokenizer/KuromojiUDF.java

```diff
@@ -69,13 +69,10 @@
     private static final int READ_TIMEOUT_MS = 60000; // 60 sec
     private static final long MAX_INPUT_STREAM_SIZE = 32L * 1024L * 1024L; // ~32MB
 
-    private Mode _mode;
```

Comment:
Any serialization problems in `_mode` and `_stopTagsArray` ?
Any problem in those early instantiation?

Reply:
As far as we can observe from the result of test case `KuromojiUDFTest#testSerialization`, there is no serialization issue here.

What this modification does is to defensively avoid early instantiation of Lucene's unclear classes (`Mode`, `CharArraySet` and `UserDictionary`).

Reply:
Mode is enum and serializable and thus better to remain the old code.